app/src/main/java/com/example/levi9application/fragments/CocktailsFragment.kt

```diff
@@ -182,12 +183,10 @@ class CocktailsFragment : Fragment(R.layout.fragment_cocktails) {
         override fun onItemClick(cocktail: Cocktail) {
```

Comment:
Current logic is that item can be added to favorites when clicking on the whole image. This should be changed so it can be added when clicking only the heart icon. Also, logic for adding items to favorites needs to be improved. Since now we have users (email field) we need to handle adding/removing items based on not only id but also email field. Investigate complex primary keys in Room db. Check Composite key and use it for this functionality.